spec/javascripts/live_search_spec.js

```diff
@@ -197,7 +180,7 @@ describe('liveSearch', function () {
 
     it('should have an order state selected when keywords are present', function () {
       liveSearch.state = 'find-eu-exit-guidance-business.json?keywords=123'
-      expect(liveSearch.$orderSelect.val()).not.toBe(null)
+      expect(liveSearch.$orderSelect.value).not.toBe(null)
```

Comment:
Weirdly even if a `select` doesn't have an empty `option`, I think it might be still possible to set the value to `null` using JS. I'm going to kick the can down the road again - I'm not comfortable removing this test based on my limited understanding of this code, and would rather look at it again in a proper refactor.

Reply:
Sure, no worry to defer it. I can open a PR afterwards.

The code doesn't actually do anything. If we change `liveSearch.state` then nothing happens except `liveSearch.state` has a different value. So any assertions on other properties aren't going to change, let alone element values.

Btw, even with JS a select.value can't be null:

```
document.querySelector('select').value
> "af07d5a5-df63-4ddc-9383-6a666845ebe9"
document.querySelector('select').value = null
> null
document.querySelector('select').value
> ""
```